### DISCUSSION TITLE
Grants, Hopes, and Dreams

### DISCUSSION THREAD

**Discussion by CECSpecialistI** [crystalyragui/MARC2RDA#431] [General]
We're considering applying for a grant to give the project a boost. We ought to discuss further:
* What are our ultimate goals for this project, specifically?
* Why does our project matter, what will it do for the library community (in words non-metadata folks can understand)?
* What do we need to accomplish our goals? If we could ask for funding for anything, what would it be?

**Comment by CECSpecialistI** [comment]
@AdamSchiff @briesenberg07 @CECSpecialistI @GordonDunsire @JianPLee @junghaelee @lake44me @SitaKB @szapoun @gerontakos @pan-zhuo

**Reply by junghaelee** [reply]
@CECSpecialistI: I added my bio to the README file.

**Comment by CECSpecialistI** [comment]
First Concrete Ask: Email Crystal a brief bio for project documentation (cec23@uw.edu) Crystal and Sofia's example bios are already in the project README file and the Wiki homepage

**Reply by GordonDunsire** [reply]
@CECSpecialistI: I added my brief bio to the README file.

**Comment by gerontakos** [comment]
I would like to address Ebe's ideas about preprocessing MARC before transforming to RDA.<br>

I'm wondering if we could write this into the grant. I would like to see several areas addressed where we anticipate the need for either pre- or post-processing, then do both. This is ambitious because it wouldn't just be two small scripts and we're done; we would need to use multiple tools: for pre-processing, XSLT-on-MARCXML, Open Refine, maybe PyMARC; for postprocessing, XSLT on RDA/RDFXML, Python on RDA/RDF-json-ld, etc. In addition, we would have to create an evaluation method, which, in itself, seems like the most computerSciency piece of this process. If I produced an evaluation method, it would be amateurish. So we'd have to identify who could do some of this work. I also don't work with all those tools. But I could produce the report/article on the results, preferably in collaboration. <br>

This would be a nice contribution of this project: I don't know how to determine what's better, pre- or post-processing. Maybe it's just "Who's on your staff and what tools do you know how to use." At any rate, this is an idea that may be too ambitious. <br>

Does anyone know if there's been work done in this area?

**Reply by corialanus** [reply]
I don't know of any specific work but I do know that people have been using Open Refine, MarcEdit and possibly other tools to clean data to be more RDA 'compliant'.

I was also thinking a bit more about this and it is interesting Theo that you mentioned various tools. With regard to the pre-processing, if possible, I think it would be useful to provide a bit more guidance on how libraries can do some of this work. Not all libraries have specialists but they can follow 'instructions' on how to look for and clean data.

I'm a believer in trying to at least do some cleaning beforehand as that will make the post cleaning work hopefully a bit simpler. Also as most people know MARC 21 well they are more comfortable in working with this data.

I realize that this project is looking at very large databases but I would like to advocate for some of the smaller libraries as well. Even if they do contribute to OCLC for example. 

Working through the two transform example, for me, was really useful in highlighting where my Library will need to look at  our data and thinking about what we can start doing now. I mentioned to a colleague the issue that came up today with regard to Place of publication. So we already have this in our sights for cleanup as soon as we can after implementing Official RDA. As she said for a solid chunk of material we can replace the various permutations with [New Zealand] which will be good enough.

Even if a small group of people who work on some of these sorts of cleanup can write up what they did and share that would be a start.

**Reply by GordonDunsire** [reply]
Pre-transform clean-up also improves the quality of the source MARC 21 data for implementation scenario C, as well as A and B. So I think this overlaps with the idea of encouraging RDA-oriented practices for MARC 21 encoding, beyond the general policies of PCC, etc. I'm thinking of aggregates, diachronic works, and collections in particular (all "new stuff").

**Reply by corialanus** [reply]
Gordon, interesting you should mention RDA-oriented practices for MARC 21. My colleague Anoushka and I were talking about that yesterday and what we thought of doing was putting together a 'best practices guide' on how to encode Official RDA in MARC 21 for scenario A. I'm also looking at implementing the collection elements (as soon as we implement) to deal with some local government documents we need to collect. I'm also looking forward to working out how, and if, Work Groups could be used for series that aren't named as such and have no numbering e.g. Harry Potter series, Artemis Fowl series but which everyone knows are a 'series'. Plus of course we will be looking at Work Groups and clustering ISSNs in diachronic works. All of these tasks will require data clean-up of some kind.
NLNZ will be implementing Official RDA from January next year.

**Comment by szapoun** [comment]
Hi everybody!
To me there are 2 major issues:
1. RDA-oriented practices, e.g., in in 6XX, 7XX fields, use $i, and also the corresponding URIs in $4.
2. Pre-transform clean-up - although at the NLG we have decided to edit/update the data before transform as much as possible.

**Reply by lake44me** [reply]
@CECSpecialistI added my bio to readme.  You are only missing Alice Chung and Cate Gerhart now.
